hawkular-command-gateway-api/src/main/resources/schema/A0uthMessage.schema.json

```diff
@@ -10,6 +10,30 @@
         "token":    { "type": "string" },
         "persona":  { "type": "string" }
       }
+    },
+    "recipientType": {
+      "type": "object",
+      "description": "A binary attachment metadata",
+      "enum": ["UI", "FEED"]
+    },
```

Comment:
this adds complexity where I don't think it is needed. What is the use-case for needing to define recipient type? first, there could be messages that could flow in either direction (for example, the simple EchoCommand, but in the future we could have Notification type commands where either a ui or feed will be given a notification message. Second, even for those messages that are destined for just one side or the other, it will be obvious by the command type itself (for example "DeployApplication" is clearly not going to have the UI as a recipient). So to me this just adds complexity where it isn't needed.